.github/workflows/spec.main.yml

```
@@ -37,13 +37,24 @@ jobs:
         with:
           name: openapi-bundled
           path: tests/openapi-bundled.yaml
-
+      - name: Set outputs
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      - name: Check outputs
+        run: echo ${{ steps.vars.outputs.sha_short }}
       - name: Upload Main spec
         run: >-
           aws s3 cp tests/openapi-bundled.yaml
           ${{ env.SPACES_PATH }}/DigitalOcean-public.v2.yaml
           --endpoint=${{ env.SPACES_ENDPOINT }}
           --acl public-read
+    # the spec that will be associated with the generated Pydo client    
+      - name: Upload Pydo spec
```

This will eventually be used for any language client, not just pydo. 

```suggestion
      - name: Upload revision spec
```